src/mcp/tools/core/init.ts

```diff
@@ -15,6 +16,16 @@ export const init = tool(
       //   .describe("Force the initialization without prompting for confirmation. Without force, it prompts if any existing files are overwritten."),
       features: z.object({
         // TODO: Add all the features here.
+        database: z.object({
+          rulesFilename: z
+            .string()
+            .default("database.rules.json")
+            .describe("The file to use for Realtime Database Security Rules."),
+          rules: z
+            .string()
+            .default(DEFAULT_RULES)
+            .describe("The security rules to use for Realtime Database Security Rules."),
```

Comment:
Make both of these `.optional()` in addition to having defaults defined.

Reply:
Sure